### ISSUE TITLE
[JavaScript] ES6 single line function expressions break highlighting

### ISSUE THREAD

**Issue by ccampbell** [sublimehq/Packages#232]
I'm not sure exactly what the scopes should be, but this breaks all highlighting after it in an ES6 file

```javascript
this._onReady.then(() => seekReady).then(() => {
    // Something here
});
```

Changing it to 

```javascript
this._onReady.then(() => { seekReady }).then(() => {
    // Something here
});
```

Makes it work correctly, but the brackets should be optional when used on a single line

**Comment by wbond** [comment]
I am not seeing this behavior with the latest version. What version are you running?

I think it should have resolved with 2231d8a7593ab58cc81353bd49327f3d46ec7703.

**Comment by ccampbell** [comment]
Yes, you are correct. I think I updated to the latest sublime dev build and removed the symlink to the JavaScript package from this repo, and it was broken there, but after updating to latest master it seems to be working correctly. Sorry for the false alarm!
